cmd/server/flags.go

```diff
@@ -473,3 +473,14 @@ var flags = append([]cli.Flag{
 		Usage:   "Flag to decrypt all encrypted data and disable encryption on server",
 	},
 }, logger.GlobalLoggerFlags...)
+
+// If woodpecker is running inside a container the default value for
+// the datasource is different from running outside a container.
+func datasourceDefaultValue() string {
+	_, found := os.LookupEnv("WOODPECKER_IN_CONTAINER")
```

Comment:
What if mainframe or Kubernetes? Two additional flags?

Reply:
The flag is only set in the Dockerfile when woodpecker-server is packaged into an image. So the same setting applies for kubernetes.
Backward compatibility.

Reply:
works for all container setups.